### DISCUSSION TITLE
How to determine if user is authenticated/logged in (any role)

### DISCUSSION THREAD

**Discussion by spaceageliving** [statamic/cms#7089] [Q&A] [answered]
This might be a dumb question, but I can't seem to find a clear answer to this:

Is there a way to determine if the user is logged in with any role?

It looks like this could be done with an elaborate {{ user:is roles="writer|editor|role-x" }} that would have to be maintained as roles are added to the system.

Is there any easier way, such as a user:is_authenticated tag? Or a way to detect if the user is a guest/not authenticated?

**Comment by joshuablum** [comment, marked as answer]
Hey @spaceageliving,

I guess the `logged_in` variable is what you're looking for? [Link to docs](https://statamic.dev/variables/logged_in)

Hope that helps :)

**Comment by spaceageliving** [comment]
@joshuablum YES....thank you, exactly what I needed.
